system/View/Plugins.php

```diff
@@ -66,6 +70,8 @@ public static function safeMailto(array $params = []): string
 
     /**
      * Wrap helper function to use as view plugin.
+     *
+     * @param list<int|string> $params
```

Comment:
```suggestion
     * @param array<int|string, string>|list<string> $params
```

When the template is `{+ lang Number.terabyteAbbr foo=1 +}`,
```php
array (
  0 => 'Number.terabyteAbbr',
  'foo' => '1',
)
```
When `{+ lang Number.terabyteAbbr foo 1 +}`,
```php
array (
  0 => 'Number.terabyteAbbr',
  1 => 'foo',
  2 => '1',
)
```

Reply:
I missed it.